ctru-sys/src/bin/docstring-to-rustdoc.rs

```diff
@@ -0,0 +1,50 @@
+use std::{env, fs, io};
```

Comment:
Could you add a short comment to the top explaining what this script is doing? I understand it from reading the code, but it would be good to have a glance-able summary.